osu.Game/Overlays/BeatmapSet/HeaderButton.cs

```diff
@@ -36,9 +36,9 @@ private void load(OsuColour colours, AudioManager audio)
             Masking = true;
             CornerRadius = 3;
             BackgroundColour = OsuColour.FromHex(@"094c5f");
-            Triangles.ColourLight = OsuColour.FromHex(@"0f7c9b");
-            Triangles.ColourDark = OsuColour.FromHex(@"094c5f");
-            Triangles.TriangleScale = 1.5f;
+            this.Triangles.ColourLight = OsuColour.FromHex(@"0f7c9b");
+            this.Triangles.ColourDark = OsuColour.FromHex(@"094c5f");
+            this.Triangles.TriangleScale = 1.5f;

```

Comment:
`Triangles` is a type.. here you try to access some properties/fields that only exists for instances of `Triangles`.
It's like writing `Person.Name` instead of doing `Person p = new Person();` `p.Name` if you understand what I mean.

Reply:
Yes but OsuButton has a definition for `protected Triangles Triangles;` that is set in OsuButtons load method. HeaderButton derives from OsuButton. This compiles and works for me... Any ideas?

Reply:
I can't see it. Triangle Button has a definition for that. OsuButton itself is empty except for a constructor. TriangleButton inherits from OsuButton like your HeaderButton does now. They have nothing in common for you to be able to access this?

Reply:
Oh wait I'm an idiot.. OsuButton.cs was updated a day ago, i think I have an old version. Should the HeaderButton derive from TriangleButton instead then?

Reply:
yep, proabbly!